packages/transport/src/core/types/PasswordProtection.ts

```diff
@@ -31,4 +32,15 @@ export class PasswordProtection extends Serializable implements IPasswordProtect
             salt: this.salt
         });
     }
+
+    private matchesCreationParameters(compare: PasswordProtectionCreationParameters): boolean {
```

Comment:
`compare` is not a good wording. wtb just `creationParameters` to match the function naming?